### PR TITLE
Feature/molecule photos uploader labels

### PR DESCRIPTION
**Pull request by AitorRodriguez990**
## Molecule/PhotoUpload
**TASK**: [SCMI-72746](https://jira.scmspain.com/browse/SCMI-72746)

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Only
- [ ] Refactor

### Description, Motivation and Context

Add a new feature to be able to add a label on the images.

### Screenshots - Animations

image

### CODE REVIEW

File: components/molecule/photoUploader/src/index.js [hunk 1]
```diff
@@ -384,7 +392,7 @@ MoleculePhotoUploader.propTypes = {
 
   /**
    *  Callback that returns an array of files.
-   *  It's executed everytime an image is added, or is deleted, or is rotated, or is sorted
+   *  It's executed everytime an image is added, or is deleted, or is rotated, or is sorted, or it's label change
```
Comment (jordevo) [inline components/molecule/photoUploader/src/index.js]:
```suggestion
   *  It's executed everytime an image is added, deleted, rotated, sorted or its label is changed
```

perhaps a bit more clear?

**Review by jordevo — APPROVED** [review]

File: components/molecule/photoUploader/demo/package.json [hunk 2]
```diff
@@ -10,6 +10,8 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "@s-ui/react-atom-button": "1"
+    "@s-ui/react-atom-button": "1",
+    "@s-ui/react-molecule-dropdown-option": "^1.42.0",
+    "@s-ui/react-molecule-select": "^1.52.0"
```
Comment (oriolpuig) [inline components/molecule/photoUploader/demo/package.json]: update the dependency version to exact.

File: components/molecule/photoUploader/src/PhotosPreview/index.js [hunk 3]
```diff
@@ -68,6 +71,26 @@ const PhotosPreview = ({
     })
   }
 
+  const _labelItem = ({index, label}) => {
+    console.log('labelItem', {index, label})
+    const list = [...files]
+    const item = list[index]
+    console.log('item', item)
```
Comment (oriolpuig) [inline components/molecule/photoUploader/src/PhotosPreview/index.js]: Should we remove the console.log()?

File: components/molecule/photoUploader/src/PhotosPreview/index.js [hunk 4]
```diff
@@ -68,6 +71,26 @@ const PhotosPreview = ({
...
+    list[index] = {
+      ...item,
+      label
+    }
+    console.log('list[index]', list)
```
Comment (oriolpuig) [inline components/molecule/photoUploader/src/PhotosPreview/index.js]: Should we remove the console.log()?

**Review by quinwacca — APPROVED** [review]

**Pull request closed without merging**